### ISSUE TITLE
Ability for mouseclick to trigger interactive layer removed?

### ISSUE THREAD

**Issue by twalts** [antonpup/Aurora#692]
### What are you experiencing an issue with?

- [X] Aurora
- [ ] Aurora supported game, (Fill in the game)
- [ ] Aurora Wrapper
- [ ] Aurora Profile
- [ ] Aurora supported device, (Fill in problematic devices)
- [ ] Aurora Scripts
- [ ] Aurora Updater
- [ ] Aurora Lighting issue
- [ ] Other, (Fill in another issue here)

### What kind of an issue is it?

- [ ] Crash
- [X] Performance problems
- [ ] Visual bug
- [ ] Localization issue
- [ ] Enhancement suggestion
- [ ] Device issue
- [ ] Other, (Fill in another type of issue here)

### Expected behavior:
When I click my mouse, the interactive layer triggers as though I pressed a key.

### Actual behavior:
There is no longer a checkbox to enable this option.

### How often does this problem occur?

This problem occurs... everytime.

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1. 
2. 
3. 

### What is the version of Aurora you are experiencing this issue with?

My version is... v0.6.0dev2

### Does the previous version of Aurora have this issue?
The previous version did not have this issue.

### Please include any logs, screenshots, or files that are related to this issue:

For logs, you can use: http://pastebin.com/
For screenshots, you can use: http://imgur.com/
For files, you can use: http://www.tinyupload.com/ , https://www.dropbox.com/ , or attach them to the issue.

**Comment by DogeAmazed** [comment]
Yes it was remove with this release: https://github.com/antonpup/Aurora/releases/tag/v0.6.0-dev
There was an performance issue with the mouse hook.

**Comment by brenden77** [comment]
Hello, wondering if this feature will ever return. I was on a beta version of 0.6.0 and didn't notice a performance issue. I upgraded to 0.7.2 and now am missing that feature. Since this was commented on in 2017, will it ever come back?
